cmd/detectExecuteScan.go

```diff
@@ -306,6 +306,12 @@ func addDetectArgs(args []string, config detectExecuteScanOptions, utils detectU
 		config.Unmap = false
 	}
 
+	if config.MinScanInterval > 0 {
+		//Unmap doesnt work well with min-scan-interval and should be removed
+		config.Unmap = false
+		args = append(args, fmt.Sprintf("--detect.blackduck.signature.scanner.arguments='--min-scan-interval=%v'", config.MinScanInterval))
```

Comment:
should we include a strict formatter since we know its a integer value `--min-scan-interval=%d` instead of `--min-scan-interval=%v`

Reply:
Agree. Will make the change